Doc/whatsnew/3.12.rst

```diff
@@ -382,6 +390,10 @@ Changes in the Python API
   to :term:`filesystem encoding and error handler`.
   Argument files should be encoded in UTF-8 instead of ANSI Codepage on Windows.
 
+* Removed the ``asyncore``-based ``smtpd`` module deprecated in Python 3.4.7
+  and 3.5.4.  A recommended replacement is the
+  :keyword:`async`/:keyword:`await`-based aiosmtpd_ PyPI module.
```

Comment:
Thanks, but can you actually say here

"... the asyncio-based ..." (with proper markup) instead?  It's a more succinct way of saying that.

(It doesn't look like I can edit the file with a suggested change.)